app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+
+
+<%= form_with model: @task, method: :patch do |f| %>
+<h2>Edit this Task</h2>
+<p> Please enter your tasks here to save them to the database:</p>
+
+<%= f.label :action %>
+<%= f.text_field :action %>
```

Comment:
You've got almost exactly the same code here to render a form as you do in `new`. You should dry this up with a view partial.